src/components/Modal/ModalFooter/ModalFooter.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import clsx from 'clsx';
+import type {ModalFooterProps} from './ModalFooter.types';
+import './ModalFooter.scss';
+
+const ModalFooterForwardRef: React.ForwardRefRenderFunction<HTMLDivElement, ModalFooterProps> = ({
+    children,
+    ...props
+}, ref) => {
+    return (
+        <div
```

Comment:
- You can use the tag `footer` to add more semantics to your markup
- Render nothing if no children